src/features/Navigation/MobileDropdown/index.tsx

```diff
@@ -37,6 +39,12 @@ const MobileDropdown: React.FC<Props> = ({ items }) => {
     i18n.changeLanguage(langCode);
   };
 
+  const unreadMessagesFound: boolean =
+    useAppSelector(selectChats)
+      .map(chat => chat.messages)
+      .flat()
+      .filter(message => !message.opened).length > 0;
```

Comment:
There is already another place with almost same selector logic. Maybe it would be worth to create a selector that already does this logic and only use that,.. like "selectHasUnreadMessages"

Reply:
Refactored!